packages/imperative/__tests__/config/cmd/init/init.handler.test.ts

```diff
@@ -634,7 +944,6 @@ describe("Configuration Initialization command handler", () => {
         (params.response.console as any).prompt = promptWithTimeoutSpy;
         writeFileSyncSpy.mockImplementation(); // Don't actually write files
 
-        jest.spyOn(process, "cwd").mockReturnValueOnce(__dirname);
```

Comment:
Curious about the reason for this change? I think the `jest.resetAllMocks` call in the beforeEach block should mean no side effects of mocks are left behind for subsequent tests.

Reply:
Good catch, this wasn't supposed to be removed. I added it back.